CODEOWNERS

```
@@ -20,6 +20,9 @@
 # OpenGL
 /src/OpenGL/**/*.cs                 @ThomasMiz @Beyley
 
+# WebGPU
+/src/WebGPU/**/*.cs @Beyley
```

(all ok but let's make it match the spacing lol)
```suggestion
/src/WebGPU/**/*.cs                 @Beyley
```